### ISSUE TITLE
Warning about deprecated sub-dep 'har-validator' when installing cypress

### ISSUE THREAD

**Issue by crfrolik** [cypress-io/cypress#8265]
### Current behavior:

```
warning cypress > @cypress/request > har-validator@5.1.5: this library is no longer supported
```

### Desired behavior:

No warning

### Test code to reproduce

`yarn install`

### Versions

Cypresss 4.12.1

**Comment by jennifer-shehane** [comment]
Replaced by https://github.com/ahmadnassri/har-cli - we'd be open to a pull request to replace this in @cypress/request here: https://github.com/cypress-io/request

**Comment by jennifer-shehane** [comment]
Related to #6730

**Comment by jennifer-shehane** [comment]
I don't really see this cli util being a 1 to 1 replacement honestly.

**Comment by bahmutov** [comment]
Kind of annoying to see it
```
$ npm i -D cypress
npm WARN deprecated har-validator@5.1.5: this library is no longer supported

> cypress@5.2.0 postinstall /Users/gleb/git/cypress-test-tiny/node_modules/cypress
> node index.js --exec install

+ cypress@5.2.0
```

**Comment by christyram99** [comment]
Hi @jennifer-shehane , I want to contribute to this issue. Can you elaborate the issue?

**Comment by christyram99** [comment]
I checked the details of the npm package and it shows it is no longer supported.

I think it should be deleted from the dependencies.

**Comment by jennifer-shehane** [comment]
```
 yarn why har-validator
yarn why v1.22.0
[1/4] 🤔  Why do we have the module "har-validator"...?

=> Found "har-validator@5.1.3"
info Has been hoisted to "har-validator"
info Reasons this module exists
   - "workspace-aggregator-c80b5799-973a-422e-82ee-a6edb3304878" depends on it
   - Hoisted from "_project_#request#har-validator"
   - Hoisted from "_project_#@cypress#request#har-validator"

=> Found "node-webkit-updater#har-validator@1.8.0"
info Reasons this module exists
   - "_project_#@packages#server#node-webkit-updater#request" depends on it
   - Hoisted from "_project_#@packages#server#node-webkit-updater#request#har-validator"
```

This PR will partially address this https://github.com/cypress-io/cypress/pull/8807 - removing the har-validator by removing the node-webkit-updater package.

@christyram99 We would need the har-validator plugin to be replaced with the same functionality in our https://github.com/cypress-io/request repo. 

Or for us to replace our https://github.com/cypress-io/request library with similar functionality in our cypress repo.

**Comment by JJ** [comment]
So I guess that PR didn't really eliminate this, right?

**Comment by mikedlr** [comment]
* take the docker image docker.io/cypress/included:8.7.0
* use --entrypoint to run a shell in the image
* run `npm install cypress --save-dev`
* read output
```
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
```
expected output: none

**Comment by mikedlr** [comment]
```
# cypress --version
Cypress package version: 8.7.0
Cypress binary version: 8.7.0
Electron version: 14.1.0
Bundled Node version: 14.17.0
```

**Comment by JJ** [comment]
Update: [eliminated here](https://github.com/cypress-io/request/pull/11) but [added back here](https://github.com/cypress-io/request/pull/15) I'd opt for eliminating the place where it's included, as well as the single sentence where it's used. I'm not totally sure that's needed any more. It's behavior carried over from the original `request` which is probably not used now (as a matter of fact, it's not tested)

**Comment by DevPerers** [comment]
I was able to install Cypress as followed, first run following prior to install Cypress
npm install --save-dev har-validator
then install cypress 
npm install cypress --save-dev

**Comment by PavelA85** [comment]
My `yarn install` fails with the message:
`Error: Cannot find module 'har-validator'`

`yarn add har-validator --dev` before `yarn add cypress  --dev` helps but does not feel right...

**Comment by JanKanty** [comment]
Shouldn't this be closed as changes were made in `@cypress/request`?

**Comment by emilyrohrbough** [comment]
@JanKanty Yes, this was resolved in [cypress-io/request#16 released ](https://github.com/cypress-io/request/pull/16) and released as [@cypress/request@v2.88.10](https://github.com/cypress-io/request/releases/tag/v2.88.10)
